app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
```

Comment:
this looks strange. you now have 2 models pointing to the same table. cannot you unify them somehow ? i think that only difference is that some comments have user and others not